masschallenge/impact-api
# [AC-6761] API endpoint for Office hours that's filterable by Users

### Changes introduced in [AC-6761](https://masschallenge.atlassian.net/browse/AC-6761):
Add an office hours endpoint filterable by mentor id and/or finalist id

### How to Test

- Run impact on this branch and visit: http://localhost:8000/api/v1/office_hours/ and use `mentor_id`, `mentor_name`, `finalist_id` and `finalist_name` as url parameters to do the filtering
- `mentor_name` and `finalist_name` should also return results for partial matches
